docs/features/custom-release-repo-paths.md

```
@@ -0,0 +1,89 @@
+
+```bash
+oc-mirror --v2 \
+  --authfile ./pull-secret.json \
+  --config ./isc.yaml \
+  --workspace file:///home/user/oc-mirror-workspace \
+  docker://registry.example.com/mirrors/quay.io
+```
+
+With the default configuration, release images would be copied to:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)